safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -985,12 +986,14 @@ def run(self):
             dict_of_analysis_summary_id[exposure_key] = (
                 impact_function.analysis_impacted.id())
 
-            # eq raster on pop raster might not having this
-            if impact_function.exposure_summary:
-                dict_of_exposure_summary_path[exposure_key] = full_layer_uri(
-                    impact_function.exposure_summary)
-                dict_of_exposure_summary_id[exposure_key] = (
-                    impact_function.exposure_summary.id())
+            # Exposure summary layer might not exist for raster on raster
```

Comment:
The exposure summary layer does not exist for: exposure continuous raster layer (whatever the hazard type).
We still have it for exposure classified raster layer.